### PR TITLE
Handle another unknown format found on omxhisi

### PR DESCRIPTION
**Pull request by turran**

### CODE REVIEW

File: sys/androidmedia/gstamc.c [hunk 1]
```diff
@@ -2110,9 +2112,12 @@ accepted_color_formats (GstAmcCodecType * type, gboolean is_encoder)
 
   for (i = 0; i < type->n_color_formats; i++) {
     gboolean found = FALSE;
+
```
Comment (dabrain34) [inline sys/androidmedia/gstamc.c:2115]: unnecessary line feed

File: sys/androidmedia/gstamc-constants.h [hunk 2]
```diff
@@ -93,13 +93,15 @@ enum
   COLOR_Format24BitARGB6666 = 42,
   COLOR_Format24BitABGR6666 = 43,
   COLOR_FormatAndroidOpaque = 0x7F000789,
+  COLOR_FormatAndroidUndocumented1 = 47,
```
Comment (dabrain34) [inline sys/androidmedia/gstamc-constants.h]: what a weired name :) Isnt it possible to document or describe it a bit ?

**Review by dabrain34 — CHANGES_REQUESTED** [review]

File: sys/androidmedia/gstamc-constants.h (hunk 2 above)
Comment (turran) [reply sys/androidmedia/gstamc-constants.h]: I agree, but on the Android documentation there's no reference for the format 47. I've searched for it but could'nt find it :-/

File: sys/androidmedia/gstamc.c (hunk 1 above)
Comment (turran) [reply sys/androidmedia/gstamc.c:2115]: Why? is the area where you declare the variables. Like in the line 2044 of the same file

File: sys/androidmedia/gstamc-constants.h (hunk 2 above)
Comment (dabrain34) [reply sys/androidmedia/gstamc-constants.h]: Can you put a comment where you explain this :)

File: sys/androidmedia/gstamc.c (hunk 1 above)
Comment (dabrain34) [reply sys/androidmedia/gstamc.c:2115]: ok

**Review by dabrain34 — CHANGES_REQUESTED** [review]

**Review by dabrain34 — APPROVED** [review]

**Pull request merged**